common/src/main/java/com/wynntils/wynn/model/ShamanMaskModel.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wynn.model;
+
+import com.wynntils.core.managers.Model;
+import com.wynntils.features.user.overlays.ShamanMasksOverlayFeature;
+import com.wynntils.mc.event.SubtitleSetTextEvent;
+import com.wynntils.mc.utils.ComponentUtils;
+import com.wynntils.wynn.event.WorldStateEvent;
+import com.wynntils.wynn.objects.ShamanMaskType;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class ShamanMaskModel extends Model {
+    private static final Pattern MASK_PATTERN = Pattern.compile("§cMask of the (Coward|Lunatic|Fanatic)");
+
+    private static ShamanMaskType currentMaskType = ShamanMaskType.NONE;
+
+    public static void init() {}
+
+    @SubscribeEvent
+    public static void onTitle(SubtitleSetTextEvent event) {
+        String title = ComponentUtils.getCoded(event.getComponent());
+
+        if (title.contains("Mask of the ") || title.contains("➤")) {
+            parseMask(title);
+
+            if (ShamanMasksOverlayFeature.INSTANCE.hideMaskTitles
+                    && ShamanMasksOverlayFeature.INSTANCE.shamanMaskOverlay.isEnabled()) {
+                event.setCanceled(true);
+            }
```

Comment:
Use an event and cancel it in the overlay itself

Reply:
Why? We do stuff like this elsewhere I am pretty sure. I guess I could duplicate the event but it would be just code copying.

Reply:
CustomBarsOverlayFeature uses an event for cancelling. This also breaks the structural barrier between features and models, which I'd like to preserve when possible.